README.md

```diff
@@ -147,14 +147,14 @@ You should be able to do the same in Mac or Windows but note that setting up a H
     [openbdre@sandbox ~]# git clone https://github.com/WiproOpenSourcePractice/openbdre.git
     ```
 
- * cd to the cloned source dir (so you can be in /home/openbdre/openbdre)
+ * `cd` to the cloned source dir (so you can be in /home/openbdre/openbdre)
 
     ```shell
     [openbdre@sandbox ~]# cd openbdre
     ```
 
 2. Database Setup
-    * Execute the dbsetup.sh script without any parameters as shown below. In this example, we are going to use MySQL as BDRE backend as it's already available in the HDP Sandbox. If you would like to use another database please select it accordingly.
+    * Execute the `dbsetup.sh` script without any parameters as shown below. In this example, we are going to use `MySQL` as BDRE backend as it's already available in the HDP Sandbox. If you would like to use another database please select it accordingly.
```

Comment:
Remove code highlight from `MySQL`. It's not a command. I think **MySQL** would be better if you feel it should be highlighted. The commands are code highlighted immediately below for users convenient. Basically what I'm thinking is when a code highlighted immediately in the next block then too much code highlight attenuate user readability.